SDL2Config.cmake.in

```diff
@@ -9,6 +9,13 @@ set_package_properties(SDL2 PROPERTIES
 @PACKAGE_INIT@
 
 
+include("${CMAKE_CURRENT_LIST_DIR}/sdlfind.cmake")
+if(@HAVE_ALSA@)
+    sdlFindALSA()
+endif()
```

Comment:
I think you need to export `SDL_ALSA` and `SDL_ALSA_SHARED` to `SDL2Config.cmake`.
`ALSA::ALSA` target will be visible on `SDL2-static` in the `INTERFACE_LINK_LIBRARIES` property when configured with `-DSDL_ALSA=ON -DSDL_ALSA_SHARED=OFF`.


Something like:
```cmake
include("${CMAKE_CURRENT_LIST_DIR}/sdlfind.cmake")
set(SDL_ALSA @SDL_ALSA@)
set(SDL_ALSA_SHARED @SDL_ALSA_SHARED@)
if(SDL_ALSA AND NOT SDL_ALSA_SHARED AND TARGET SDL2::SDL2-static)
    sdlFindALSA()
endif()
unset(SDL_ALSA_SHARED)
unset(SDL_ALSA)
```
(I'm unsetting these variables because these are not part of the current SDL2 CMake API (and hard to port to autotools).

Add this block below the includes where the `SDL2::*` targets are created.